### PR TITLE
Flask support

### PR DESCRIPTION
**Pull request by jezdez**
This adds support for Flask and fixes a bunch of things in the Django support as well.

This is still missing docs sadly.

### CODE REVIEW

**Comment by jezdez** [comment]
@robotblake Any chance you taking a look at this? Or anyone else interested in Dockerflow? @mythmon maybe?

**Comment by jezdez** [comment]
Hey @haroldwoo, I just pushed a bunch of updates and a lot of docs here, are you still up for taking a look at this?

File: docs/flask.rst [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+If you'd like to override the location from which the view is reading the
+``version.json`` file from, simply override the optional ``version_path``
+parameter to the :class:`~dockerflow.flask.app.Dockerflow` class, e.g.::
+
+    from flask import Flask
+    from dockerflow.flask import Dockerflow
+
+    app = Flask(__name__)
+    dockerflow = Dockerflow(app, version_path='/app')
```
Comment (haroldwoo) [inline docs/flask.rst]: thanks for this

File: docs/flask.rst [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+        return json.loads(os.path.join(root, 'acme_version.json'))
+
+.. _version object: https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md
+
+.. _flask-health:
+
+Health
+------
+
+TODO
```
Comment (haroldwoo) [inline docs/flask.rst]: I am not sure how to set up lbhealthchecks?

File: tests/settings.py [hunk 3]
```diff
@@ -39,6 +39,7 @@
 
 SILENCED_SYSTEM_CHECKS = [
     'security.W002',
+    'security.W003',
```
Comment (haroldwoo) [inline tests/settings.py]: nit order

File: docs/flask.rst (hunk 2 above)
Comment (haroldwoo) [reply docs/flask.rst]: it looks like the healthcheck endpoint runs through configured checks and the lbhealthcheck simply returns 200 if the port is responding?

**Comment by haroldwoo** [comment]
This PR looks good to me, though you may want to get another to review as well.  I'm assuming the tests are ran by tox/travis and I don't have experience with the alembic sqlalchemy stuff.  Thanks for doing this!

**Comment by jezdez** [comment]
@washort r?

**Comment by washort** [comment]
r+

File: tests/settings.py (hunk 3 above)
Comment (jezdez) [reply tests/settings.py]: 👍

**Pull request merged**